contract/contracts/NewInterfaces.sol

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.6.0;
+
+contract DistributerInterface {
+    // TODO Add events
+
+    string name;
+    address tokenHolder;
+    mapping(address => address) tokenRefundDestinationList;
+
+    function balanceOf(address token) virtual external view returns (uint256) {}
+
+    function addToken() virtual external {}
+
+    function createCampaign(
+        address token,
+        string memory campaignInfoCid,
+        string memory recipientsCid,
+        string memory recipientsNum,
```

Comment:
What do you think about the possibility of a discrepancy between the number of elements in recipientsCid and the number in recipientsNum?

Reply:
`recipientsCid` was for getting campaign recipients but it achieved by getting elements of JSON elements from IPFS. 
So I deleted that variable.